src/test/components/formComponents/sourceWizardSummary.test.js

```diff
@@ -287,7 +287,8 @@ describe('SourceWizardSummary component', () => {
         ['Source type', 'Google Cloud'],
         ['Application', 'Cost Management'],
         ['Project ID', 'project_id_123'],
-        ['Dataset ID', 'dataset_id_123'],
+        ['', '-'],
```

Comment:
We should probably filter this not to be displayed in a review step using `hideInReview` prop